src/io/flutter/module/InstallSdkAction.java

```diff
@@ -143,6 +145,7 @@ public void validateSelectedFiles(VirtualFile[] files) throws Exception {
 
       final VirtualFile installTarget = FileChooser.chooseFile(descriptor, null, null);
       if (installTarget != null) {
+        FlutterInitializer.sendAnalyticsAction(getClass().getSimpleName());
```

Comment:
This is sending `GitCloneAction `; do we need to do that? Is this always called when `InstallSdkAction` runs?